packages/ci/test/commands/ci/index.test.ts

```diff
@@ -1,17 +1,16 @@
 import Nock from '@fancy-test/nock'
 import * as Test from '@oclif/test'
-
 const test = Test.test
-.register('nock', Nock)
+  .register('nock', Nock)
 const expect = Test.expect
 
 describe('ci', () => {
   test
-  .command(['ci'])
-  .catch(e => {
-    expect(e.message).to.contain('Required flag:  --pipeline PIPELINE or --app APP')
-  })
-  .it('errors when not specifying a pipeline or an app')
+    .command(['ci'])
```

Comment:
@talbright please remove these indentations throughout the tests it should look like:
```
    test
    .stdout()
    .nock('https://api.heroku.com', api => {
      api.get(`/pipelines?eq[name]=${pipeline.name}`)
```

Reply:
UPDATE: we ended up deciding TO include this whitespace so it can be left in. I'll wait to merge my PR that enforces this style until after this is merged though